Made My Website Better!

AMora28 opened the pull request.

AMora28 committed:
Made My Website Better!

sampoder commented on week2/alejandro_mora/Website Structure.txt:
Could you help me understand this file? Thanks!

AMora28 replied:
Hi! The text file I created was made purely to write down ideas for what I could implement into my website.

sampoder reviewed (changes requested):
See above ^

AMora28 committed:
Created Kitty Game!

I am currently in the midst of making a "Fade in" function, where the cat will slowly appear onto the screen.

AMora28 committed:
Made a Fade in feature

My kitty now fades in once a button is pressed. Working on syncing audio to it when the button is pressed as well.

AMora28 committed:
Merge branch 'cubstart:main' into main

sampoder reviewed (changes requested):
Could you make it `week4/alejandro_mora/` and not `week4/week3/alejandro_mora/`?

AMora28 committed:
Fixed week4 repository name

AMora28 committed:
Merge branch 'main' of https://github.com/AMora28/spring25

AMora28 committed:
Fixed Naming

AMora28 committed:
Created Chatbot!

AMora28 committed:
Merge branch 'cubstart:main' into main

AMora28 committed:
Week 7/8 Work

AMora28 committed:
Created Shopping List

AMora28 committed:
Fixed SL function signature error

AMora28 committed:
Final Project!

Copied over final project files from personal computer onto Cubstart repository. Project not completely finished.
